src/Features/Core/Diagnostics/HostAnalyzerManager.cs

```diff
@@ -348,7 +353,7 @@ private void UpdateCompilerAnalyzerMapIfNeeded(string language, ImmutableArray<D
 
         private static string GetAnalyzerReferenceId(AnalyzerReference reference)
         {
-            return reference.Display ?? FeaturesResources.Unknown;
+            return reference.Id;
```

Comment:
Thank you! Probably don't need the method `GetAnalyzerReferenceId` anymore as Id is property on AnalyzerReference itself.